samples/word/02-paragraphs/get-paragraph-on-insertion-point.yaml

```diff
@@ -7,63 +7,71 @@ api_set:
     WordApi: 1.1
 script:
     content: |-
-        $("#get-paragraph").click(getParagraph);
-        $("#get-sentences").click(getSentences);
-        $("#setup").click(setup);
+        $("#get-paragraph").click(() => tryCatch(getParagraph));
+        $("#get-sentences").click(() => tryCatch(getSentences));
+        $("#setup").click(() => tryCatch(setup));
 
 
-        function getParagraph() {
-            Word.run(function (context) {
+        async function getParagraph() {
+            await Word.run(async (context) => {
                 // the collection of paragraphs of the current selection returns the full paragraphs contanied on it. 
-                var paragraph = context.document.getSelection().paragraphs.getFirst();
+                let paragraph = context.document.getSelection().paragraphs.getFirst();
+                paragraph.load("text");
                 context.load(paragraph);
-                return context.sync()
-                    .then(function () {
-                        console.log(paragraph.text);
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+
+                await context.sync();
+                console.log(paragraph.text);
+            });
         }
 
-        function getSentences() {
-            Word.run(function (context) {
+        async function getSentences() {
+            await Word.run(async (context) => {
                 // gets the complete sentence  (as range) associated with the insertion point.
-                var sentences = context.document
+                let sentences = context.document
                     .getSelection().getTextRanges(["."] /* Using the "." as delimiter */, false /*means without trimming spaces*/);
-                context.load(sentences);
-                return context.sync()
-                    .then(function () {
-                        //  expands the range to the end of the paragraph to get all the complete sentences.
```

Comment:
I notice that you're not consistent in capitalizing the first letter of the first word of a code comment. I think our recommend style is to capitalize it. So this would be "// Expands ..."

Reply:
fixed the entire script.